<launch>

    <!--<include file="$(find mavros)/launch/px4.launch" >
        <arg name="fcu_url" value="udp://:14540@127.0.0.1:14557"/> </include>-->

    <!--<node pkg="offb_posctl" type="uavinstance.py" name="uavinstance" > </node>-->
    <node pkg="offb_posctl" type="offb_posctl" name="offb_posctl" output="screen"> </node>
<!--    <node pkg="offb_posctl" type="acado_lag_control" name="acado_lag_control" output="screen"> </node>-->
    <node pkg="offb_posctl" type="bvp_control.py" name="bvp_control" > </node>
<!--    <node pkg="offb_posctl" type="bvp_control_table.py" name="bvp_control_table" > </node>-->
<!--    <node pkg="offb_posctl" type="bvptest.py" name="bvptest" > </node>-->

    <!--<node pkg="rqt_graph" type="rqt_graph" name="rqt_graph"> </node>-->

</launch>
